continue-reading-style.css

```diff
@@ -1,396 +0,0 @@
-  grid-template-areas:
-    'learn-more-title         .                          '
-    'learn-more-content-block    learn-more-content-block'
-    'learn-more-button        .                          ';
-}
-
-.learn-more-title {
-  display: flex;
-  grid-area: learn-more-title;
-  color: gray;
-  font-size: 20px;
-  font-family: 'Courier New', Courier, monospace;
-  font-weight: 600;
-  text-shadow: 0.5px 0.5px 2px black;
-}
-
-.learn-more-content-block {
-  grid-area: learn-more-content-block;
-}
-
-.learn-more-img {
-  align-self: flex-start;
-  float: left;
-  object-fit: contain;
-  padding-right: 2rem;
-  padding-bottom: 1rem;
-}
-
-.learn-more-button {
-  display: flex;
-  grid-area: learn-more-button;
-  justify-content: space-evenly;
-  align-items: center;
-  color: blanchedalmond;
-  font-family: 'Courier New', Courier, monospace;
-  font-size: 2rem;
-  font-variant: small-caps;
-  text-shadow: 0.5px 0.5px 2px grey;
-  background-image: linear-gradient(to right, #fc4445, #fc4446, #fc4447);
-  border-radius: 5px;
-  box-shadow: 1px 1px 3px gray;
-  font-weight: 600;
-  margin-right: 5rem;
-}
-
-.learn-more-button:active {
-  background-image: none;
-  background-color: grey;
-}
-
-.learn-more-button-img {
-  height: 1rem;
-  width: 1rem;
-}
-
-/* Main Content Container ***************************************************************************************************/
-
-#main-content {
-  display: grid;
-  grid-area: main-content;
-  grid-template-rows: 2rem 20rem 1fr;
-  grid-template-areas:
-    'main-content-title'
-    'main-content-img'
-    'main-content-paragraphs';
-  margin-top: 3rem;
-  gap: 3rem;
-}
-
-#main-content-title {
-  display: flex;
-  grid-area: main-content-title;
-  color: gray;
-  text-shadow: 1px 1px 2px black;
-  font-variant-caps: small-caps;
-  font-size: 2rem;
-  font-family: 'Courier New', Courier, monospace;
-  font-weight: 600;
-}
-
-#main-content-img {
-  display: flex;
-  grid-area: main-content-img;
-  object-fit: contain;
-  box-shadow: 5px 5px 8px grey;
-}
-
-/* Main Content Paragraph Blocks --------------------------------------------------------------------------------------------------*/
-#main-content-paragraphs {
-  display: grid;
-  grid-area: main-content-paragraphs;
-  grid-template-rows: 20rem 20rem 20rem 20rem;
-  grid-template-areas:
-    'main-content-block-one'
-    'main-content-block-two'
-    'main-content-block-three'
-    'main-content-block-four'
-    'main-content-block-five';
-  gap: 1rem 1rem;
-}
-
-.main-content-block {
-  display: grid;
-  grid-template-rows: 2rem 18rem;
-}
-
-.main-content-block p {
-  font-size: 15px;
-}
-
-.main-content-subtitle {
-  display: flex;
-  color: gray;
-  font-size: 20px;
-  font-family: 'Courier New', Courier, monospace;
-  font-weight: 600;
-  text-shadow: 0.5px 0.5px 2px black;
-}
-
-/* Footer /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
```

Comment:
If you have both a generic css file for elements that are shared across pages, and page specific css files you can include both, and then only have to write out the repeated styles once.